### PR TITLE
[Enhance] Modify readme

### PR DESCRIPTION
**Pull request by xiliu8006**
Modify README, getting_started and CONTRIBUTING

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,26 @@ Please refer to [changelog.md](docs/changelog.md) for details and release histor
 Supported methods and backbones are shown in the below table.
 Results and models are available in the [model zoo](docs/model_zoo.md).
 
+Support backbones:
+
+- [x] PointNet (CVPR'2017)
+- [x] PointNet++ (NeurIPS'2017)
+- [x] RegNet (CVPR'2020)
+- [x] MultiBackbone(ECCV'2020)
```
Comment (ZwwWayne) [inline README.md:74]: MultiBackbone used in H3DNet

File: README.md [hunk 2]
```diff
@@ -66,6 +66,26 @@ Please refer to [changelog.md](docs/changelog.md) for details and release histor
...
+
+- [x] [SECOND (Sensor'2018)](configs/second/README.md)
+- [x] [PointPillars (CVPR'2019)](configs/pointpillars/README.md)
+- [x] [FreeAnchor (NeurIPS'2019)](configs/free_anchor/README.md)
+- [x] [VoteNet (ICCV'2019)](configs/votenet/README.md)
+- [x] [H3DNet (ECCV'2020)](configs/h3dnet/README.md)
+- [x] [3DSSD (CVPR'2020)](configs/3dssd/README.md)
+- [x] [Part-A2 (TPAMI'2020)](configs/parta2/README.md)
+- [x] [MVXNet (ICRA'2019)](configs/mvxnet/README.md)
+- [x] [CenterPoint (arxiv'2020)](configs/centerpoint/README.md)
```
Comment (ZwwWayne) [inline README.md:86]: Arxiv

File: docs/getting_started.md [hunk 3]
```diff
@@ -177,7 +192,7 @@ PYTHONPATH="$(dirname $0)/..":$PYTHONPATH
 
 ### Point cloud demo
 
-We provide a demo script to test a single sample.
+We provide a demo script and to test a single sample, and if you want to use our pre-trained model, you can download them from [model zoo](model_zoo.md)
```
Comment (ZwwWayne) [inline docs/getting_started.md:195]: check grammar issues.

**Review by Tai-Wang — APPROVED** [review]
Just modify according to current comments.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**